src/Command/AddMassMediaCommand.php

```diff
@@ -48,6 +48,8 @@ public function __construct(ManagerInterface $mediaManager)
 
     protected function configure(): void
     {
+        \assert(null !== static::$defaultDescription);
```

Comment:
I think it is declared on symfony side as nullable

Reply:
I don't think it can be null since you set it, never set it again, the property is protected and the class is final ?
But this might be too complicated for phpstan for a feature request

Reply:
https://github.com/symfony/console/blob/5.3/Command/Command.php#L43-L46

Reply:
Not sure it will ever be just a string, because there are multiple ways to set the description (Attributes on php8, tags on the dependency injection, the static property and just calling setDescription)